pkg/keychain/keychain.go

```diff
@@ -157,15 +155,6 @@ func GetKeychainFromCmdLineFlags(
 				return nil, err
 			}
 		}
-	case network.Kind == models.Mainnet:
-		// mainnet requires ledger usage
-		if keyName != "" {
```

Comment:
do we need to keep this check?

Reply:
we don't need. this PR enables using stored keys on mainnet